fpsim/sim.py

```diff
@@ -798,6 +806,7 @@ def init_step_results(self):
             pp12to23        = 0,
             total_women_fecund = 0,
             pregnancies     = 0,
+            short_interval_age     = [],
```

Comment:
Reviewing structure that was added while I was out, it looks like self.pars['track_as'] was added to add these types of adverse outcomes over time.  We can use that structure to add this in the same adverse outcome format

Reply:
On more review, it seems like there's a bit of new structure around plotting age_specific adverse outcomes.  I need to review how this works more as I think we can likely use that structure to plot

Reply:
A bit = A LOT.  The total number of live births in an age-specific bin is already being tracked every timestep with this new structure with the 'mmr_denominator" construct.  We can use this to calculate the percent of short interval births of all live births within that age bin.  I think we should adjust the age bins for this though to be much wider than they currently are: 

method_youth_age_map = {
    '<16': [10, 16],
    '16-17': [16, 18],
    '18-19': [18, 20],
    '20-22': [20, 23],
    '23-25': [23, 26],
    '>25': [26, max_age+1]
}